pandas/core/groupby/groupby.py

```diff
@@ -402,15 +402,22 @@ class providing the base-class of operations.
 f : function, str
     Function to apply to each group. See the Notes section below for requirements.
 
-    Can also accept a Numba JIT function with
-    ``engine='numba'`` specified.
+    Accepted inputs are:
+
+    - String function name
```

Comment:
I think we can just say "String" here; you specify below (L418-420) the requirements of the string.

Reply:
Alright, thanks for the feedback. I have updated the docs here: 04d48f285b9019c71ba77e52170dc69e3c636eee